run.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
-set -ex
+
+if [ "$1" == "debug" ]; then
+    export DEBUG=1
+fi
 
 if [ "$1" == "index" ]; then
     MAIN="main_index"
     PORT=5001
+    if [ "$2" == "debug" ]; then
+        export DEBUG=1
+    fi
 else
     if [ "${MAIN}" == "main_index" ]; then
         PORT=5001
```

Comment:
Do you think it would be helpful for debug-mode to set the log-level of uvicorn to debug too (`--log-level debug`)?

Reply:
Maybe, yes. I can see what will happen

Reply:
I tested it locally and couldn't really get it to give me any more output (after startup) at least, so its no big deal!